pkg/k8s/apis/cilium.io/v2alpha1/types.go

```diff
@@ -82,3 +84,52 @@ type EgressRule struct {
 	// selector semantics; if present but empty, it selects all pods.
 	PodSelector *slimv1.LabelSelector `json:"podSelector,omitempty"`
 }
+
+// CoreCiliumEndpoint is slim version of status of CiliumEndpoint.
```

Comment:
I assume the plan is for this structure to be a subset of the CiliumEndpoint

Reply:
I'm nor sure that is the plan, As per my understanding we are going to keep it separate from CiliumEndpoint.
@Weil0ng can you confirm ?

Reply:
In theory it is a subset of CiliumEndpoint, but plan is to keep this stable (even as CEP grows with more debug/local info in the future).

Reply:
Is the goal to embed this structure in CEP?

Reply:
At this point, CoreCEP would be standalone object. We are not planning to embed with CEP.